src/wattsi.pas

```diff
@@ -904,6 +904,63 @@    TCrossReferences = record
       end;
    end;
 
+   function IsMDNBox(const Element: TElement): Boolean;
```

Comment:
This isn't used; should it be?

Reply:
It’s not used — thanks for catching that, I’ll remove it